eng/pipelines/templates/steps/build.yml

```diff
@@ -48,12 +48,29 @@ steps:
   - template: /eng/pipelines/templates/steps/set-artifact-packages.yml
     parameters:
       Artifacts: ${{ parameters.Artifacts }}
+  - ${{if and(in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
```

Comment:
It is probably better to only include this in js - core - ci, instead of every service directory's CI.

Reply:
Actually, it may be fine, whichever package finishes building first will store to the cache and subsequent builds will read from there.

We probably want to remove "Manual", or use "rush rebuild" command in releasing builds to not read from cache.